srcpkgs/lapce/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'lapce'
+pkgname=lapce
+version=0.2.0
+revision=1
+build_style=cargo
+hostmakedepends="cargo python cmake pkg-config"
+makedepends="expat fontconfig-devel freetype gtk+3-devel libxcb openssl-devel glib-devel cairo-devel atk-devel gdk-pixbuf-devel"
+short_desc="Lighting-fast and powerful code editor"
+maintainer="Simon <simon.larsson-general@protonmail.com>"
+license="Apache-2.0"
+homepage="https://lapce.dev"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/v${version}/$pkgname-$pkgver.tar.gz"
```

Comment:
what is pkgver supposed to be?

Reply:
oops, supposed to be `version` but I'm surprised it still worked. Seems like github allows any filename as long as it ends with `.tar.gz` and it'll redirect to the right one